Tools/make-pkgconfig.sh

```diff
@@ -20,7 +20,7 @@ echo Name: LLVM > $filename
 echo Description: Low-level Virtual Machine compiler framework >> $filename
 echo Version: $(echo ${version} | sed 's/\([0-9.]\+\).*/\1/') >> $filename
 echo URL: http://www.llvm.org/ >> $filename
-echo Libs: ${libs} -L$(llvm-config --libdir --system-libs --libs analysis bitwriter core native passes target) >> $filename
+echo Libs: -L$(llvm-config --libdir) $(pkg-config --libs-only-L libzstd) ${libs} $(llvm-config --system-libs --libs analysis bitwriter core native passes target) >> $filename
```

Comment:
Almost surprised to see we you didn't have to write a per-os logic here. But happy to see it works on all 3 platforms.